.cirrus.yml

```diff
@@ -132,3 +141,18 @@ promote_task:
     folder: ${CIRRUS_WORKING_DIR}/.m2/repository
   script: cirrus_promote_maven
   cleanup_before_cache_script: cleanup_maven_repository
+
+build_and_release_sonar_secrets_dotnet_task:
+  depends_on:
+    - promote
+  <<: *ONLY_IF_SONARSOURCE_QA
+  <<: *WINDOWS_VM_DEFINITION
+  env:
+    PROJECT_DIR: ${CIRRUS_WORKING_DIR}\\sonar-text-dotnet
+    SOLUTION_DIR: ${PROJECT_DIR}\\SonarLint.Secrets.DotNet.sln
+    PIPELINE_SCRIPTS_DIR: ${PROJECT_DIR}\\pipeline_scripts
+  restore_and_build_script:
+    - source cirrus-env BUILD # sets the $BUILD_NUMBER that is needed for the project version
```

Comment:
I had to read a bit to see how source works and what's actually doing :). I found this helpful: https://ss64.com/bash/source.html

Do you know where `cirrus-env` is coming from?

Reply:
Yes, it's [here](https://github.com/SonarSource/re-ci-images/blob/master/docker/bin/cirrus-env#L28).

Reply:
Cool. Thanks!